tests/conftest.py

```diff
@@ -159,20 +158,21 @@ def sample_pv_batch():
     return torch.load("tests/test_data/presaved_batches/train/000000.pt")
 
 
-@pytest.fixture()
-def sample_wind_batch():
-    dm = WindDataModule(
-        configuration=None,
-        batch_size=2,
-        num_workers=0,
-        prefetch_factor=None,
-        train_period=[None, None],
-        val_period=[None, None],
-        test_period=[None, None],
-        batch_dir="tests/test_data/sample_wind_batches",
-    )
-    batch = next(iter(dm.train_dataloader()))
-    return batch
+# TODO update this test once we add the loading logic for the Site dataset
```

Comment:
I cant work out if we should merge before doing this, im not sure there. Relates to https://github.com/openclimatefix/ocf-data-sampler/issues/99